lld/ELF/Relocations.cpp

```diff
@@ -2367,7 +2367,63 @@ void elf::hexagonTLSSymbolUpdate(ArrayRef<OutputSection *> outputSections) {
       });
 }
 
+static bool matchesRefTo(const NoCrossRefCommand &cmd, StringRef osec) {
+  if (cmd.toFirst)
+    return cmd.outputSections[0] == osec;
+  return llvm::is_contained(cmd.outputSections, osec);
+}
+
+template <class ELFT, class Rels>
+static void scanCrossRefs(const NoCrossRefCommand &cmd, OutputSection *osec,
+                          InputSection *sec, Rels rels) {
+  for (const auto &r : rels) {
+    Symbol &sym = sec->file->getSymbol(r.getSymbol(config->isMips64EL));
+    // The destination output section can be nullptr, osec, or those described
```

Comment:
I'm don't fully understand what 
``` or those described by the NOCROSSREFS/NOCROSSREFS_TO command.``` means in this context.

If the comment is describing permitted cross-references then I'd expect something like:
```A legal cross-reference is when the destination output section is nullptr, osec for a self-reference, or a section that is not in the set described by the NOCROSSREFS/NOCROSSREFS_TO command.```

Reply:
Thanks for the suggestion. Improved the comment.